R/functions.R

```diff
@@ -159,16 +159,17 @@ input_heatmap = function(.data,
 	new(
 		"InputHeatmap",
 		data = .data %>% reduce_to_tbl_if_in_class_chain,
-		input = list(
```

Comment:
Thanks. Can you please comment (in the code as well) on what this contraption is practically doing?

How come row_names_gp are not in your addition?

Reply:
Due to the `.homonyms="last"` parameter, additional arguments passed by the user via `...` overwrite the defaults given below (See also `?rlang::dots_list`). 

So if the user specifies any of the parameters you already have, the ones by the user take precedence. If the user doesn't specify anything, the behaviour is unchanged.

Reply:
Quite elegant.

But why 
```
column_title =
...
row_names_go =
```
are not alongside `column_names_gp` parameter?

Reply:
I'm not sure if I get it, but maybe it's just how the diff is visualized? 
I haven't changed the `column_names_gp` line, except for the comma.

Reply:
My bad. Sorry late at night. I will accept if the test pass.

Thanks.